toolchains/msvc.toolchain.cmake

```diff
@@ -26,20 +28,25 @@ set( TNUN_compiler_exceptions_off        -wd4577
 set( TNUN_compiler_report_optimization   -Qpar-report:1 -Qvec-report:2               )
 set( TNUN_compiler_optimize_for_speed    -Ox -Ot -Qpar                               ) # https://msdn.microsoft.com/en-us/library/jj658585.aspx Vectorizer and Parallelizer Messages
 set( TNUN_compiler_runtime_sanity_checks -GS -sdl -guard:cf -fp:strict -RTC1 -RTCc -D_ALLOW_RTCc_IN_STL ) # https://www.reddit.com/r/cpp/comments/46mhne/rtcc_rejects_conformant_code_with_visual_c_2015
+set( TNUN_disabled_warnings              -wd4324 -wd4373 -wd4100 -wd4456 -wd4067 -wd4995 -wd4635 -wd4800 -wd4101 -wd4756 -wd4068 -wd4056 -wd4996 -wd4503 )
+set( TNUN_warnings_as_errors             -WX )
+set( TNUN_default_warnings               -W4 )
 
-set( TNUN_compiler_release_flags -DNDEBUG -Bt -Ox -Ob2 -Oy -GF -Gw -Gm- -GS- -Gy )
 
 # w4373: '...': virtual function overrides '...', previous versions of the compiler did not override when parameters only differed by const/volatile qualifiers
 # w4324: 'structure was padded due to alignment specifier'
-add_compile_options( /std:c++latest -MP -Oi -W4 -Zc:threadSafeInit- -wd4324 -wd4373 -wd4100 -wd4456 -wd4067 -wd4995 -wd4635 -wd4800 -wd4101 -wd4756 -wd4068 -wd4056 -wd4996 -wd4503)
+add_compile_options( /std:c++latest -MP -Oi -Zc:threadSafeInit- )
 add_definitions(
   -D_CRT_SECURE_NO_WARNINGS
   -D_SCL_SECURE_NO_WARNINGS
   -D_SBCS
   -D_WIN32_WINNT=0x0601 # Win7
+  -DNO_STRICT
```

Comment:
this is known to break some software/libraries that depend on these...but ok let's leave it like that for now..